eventhubs-streamanalytics-eventhubs/tools/README.md

```diff
@@ -1,11 +1,41 @@
-# Testing End-To-End Latency
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure eventhubs 
+extensions:
+- platforms: dotnet
+---
 
-Get the Event Hub connection string created and displayed during the script execution. If you cannot find it, just grab the execution string for the `Listen` Shared Access Policy for the output eventhub.
+# Getting started on processing received message from eventhub #
 
+Azure eventhubs-streamanalytics-eventhubs sample for processing received message from eventhub -
+ - Create an EventHubConsumerClient based on connection string for the `Listen` or `Manage` Shared Access Policy for the eventhub.
```

Comment:
We typically demonstrate an Event Hubs namespace-level connection string, due to how much difficulty there is in finding and creating one for a specific Event Hub instance.  Personally, I'd recommend using the namespace-level here.  That said, if you're set on keeping the latter, I'd recommend either detailing how to create/find that Event Hub-level connection string, either here directly or by linking to some documentation.

Reply:
As this part is a describing part, some describing words have been updated. And i have added detailing how to create/find that Event Hub-level connection string in "Running this Sample" part.